Packs/FeedNVDv2/Integrations/FeedNVDv2/FeedNVDv2.py

```diff
@@ -0,0 +1,421 @@
+from dateparser import parse
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+DATE_FORMAT = "%Y-%m-%dT%H:%M:%S.000"  # ISO8601 format with UTC, default in XSOAR
+
+
+class Client(BaseClient):
+    """Client class to interact with the service API
+    """
+
+    def __init__(self, base_url: str, verify: bool, proxy: bool,
+                 api_key: str, tlp_color: str, has_kev: bool, start_date: str,
+                 feed_tags: list[str]):
+        super().__init__(base_url=base_url, verify=verify, proxy=proxy)
+        self._base_url = base_url
+        self.tlp_color = tlp_color
+        self.verify = verify
+        self.proxy = proxy
+        self.api_key = api_key
+        self.has_kev = has_kev
+        self.feed_tags = feed_tags
+        self.start_date = start_date
+
+    def http_get(self, path: str, params: dict):  # pragma: no cover
+        """
+        Perform a basic HTTP call using specified headers and parameters
+        """
+
+        headers = {
+            'apiKey': self.api_key
+        }
+
+        full_url = self._base_url + path
+
+        try:
+            resp = self._http_request('GET', full_url=full_url, headers=headers, params=params, resp_type='json', timeout=300)
+        except Exception as e:
+            demisto.debug(e)
+
+        return resp
+
+
+def build_indicators(client: Client, raw_cves: List):
+    """
+    Iteratively processes the retrieved CVEs from retrieve_cves function
+    and parses the returned JSON into the required XSOAR data structure
+
+    Args:
+        cve_list: CVEs retrieved using the retrieve_cves function
+
+    Returns:
+        None
+
+    """
+
+    indicators = []
+
+    for raw_cve in raw_cves:
+        indicator: dict[Any, Any] = {}
+        cvss_metric = ""
+        metrics: List = []
+        cpes: List = []
+        refs: List = []
+
+        indicator = {"value": raw_cve.get('cve').get('id')}
+
+        fields = {"description": raw_cve.get('cve').get('descriptions')[0].get('value')}
+        fields["cvemodified"] = raw_cve.get('cve').get('lastModified')
+        fields["published"] = raw_cve.get('cve').get('published')
+        fields["updateddate"] = raw_cve.get('cve').get('lastModified')
+        fields["vulnerabilities"] = raw_cve.get('cve').get('weaknesses')
+
+        # Process references
+        if len(raw_cve.get('cve').get('references')) > 1:
+            for ref in raw_cve.get('cve').get('references'):
+                url = ref.get('url')
+                source = ref.get('source')
+                refs.append({'title': indicator['value'], 'source': source, 'link': url})
+        elif len(raw_cve.get('cve').get('references')) == 1:
+            url = raw_cve.get('cve').get('references')[0].get('url')
+            source = raw_cve.get('cve').get('references')[0].get('source')
+            refs.append({'title': indicator['value'], 'source': source, 'link': url})
+        fields["publications"] = refs
+
+        # Process CPEs
```

Comment:
Should also be a function.

If you give it a title, and it does stuff and returns a value - its a function :)